server/src/recipe/recipe.controller.js

```diff
@@ -0,0 +1,18 @@
+import { OK } from 'http-status';
+import { Recipe } from '@/shared/database/index';
+
+const list = async (req, res, next) => {
+  try {
+    const recipes = await Recipe.findAll({
+      where: {
+        isPublished: true,
+      },
+    });
+    return res.status(OK).json(recipes);
+  } catch (err) {
+    console.log(err);
+    return next(new Error());
```

Comment:
Use the not found error here but check first if it is an instance of not found error